.github/workflows/readiness_comment.yml

```
@@ -0,0 +1,177 @@
+          * If this pull request is already a separate pull request from the above item, you can ignore this message.
+          </details>
+          ' >> note.md
+
+      - name: Changelog Updated by Non-Maintainers
+        id: changelog
+        if: |
+          steps.filter.outputs.changelog == 'true'
+          && steps.community_check.outputs.maintainer == 'false'
+        shell: bash
+        run: |
+          echo '<details><summary>:x: Unnecessary Changelog Changes (Click to expand)</summary><br>
+
+          The `CHANGELOG.md` file contents are handled by the maintainers during merge. This is to prevent pull request merge conflicts, especially for contributions which may not be merged immediately. Please see the [Changelog Process](https://hashicorp.github.io/terraform-provider-aws/changelog-process/) section of the contributing guide for additional information.
+
+          Remove any changes to the `CHANGELOG.md` file and commit them in this pull request to prevent delays with reviewing and potentially merging it.
+          </details>
+          ' >> note.md
+
+      - name: Needs Changelog Entry
+        id: needs-changelog-entry
+        if: |
+          steps.changelog-update-needed.outputs.needs == 'true'
+          && steps.changelog-update-needed.outputs.has == 'false'
+          && steps.community_check.outputs.maintainer == 'false'
+        shell: bash
+        run: |
+          echo '<details><summary>:x: Changelog Entry Required (Click to expand)</summary><br>
+
+          The proposed change requires a changelog entry. Please see the [Changelog Process](https://hashicorp.github.io/terraform-provider-aws/changelog-process/) section of the contributing guide for information on the changelog generation process.
+
+          **Tip:** This check is not triggered for draft pull requests, since the pull request number is not known until the pull request is opened and is required to create a changelog entry. Opening a pull request first as a draft, adding the requisite changelog entry file, and then marking the pull request as ready for review will prevent future warnings.
+           </details>
+          ' >> note.md
+
+      - name: Add Start Message
+        id: start
+        if: |
+          steps.maintainer_editability.outcome != 'skipped'
+          || steps.dependencies.outcome != 'skipped'
+          || steps.changelog.outcome != 'skipped'
+          || steps.needs-changelog-entry.outcome != 'skipped'
+        shell: bash
+        run: |
+          { echo $START_TEXT; echo "---"; cat note.md; } > tmpnote && mv tmpnote note.md
```

Could this write directly to `note.md` instead of the `mv` step?

```suggestion
          { echo $START_TEXT; echo "---"; cat note.md; } > note.md
```

---

Ah - I missed that `note.md` was referenced when constructing the text. This approach seems fine if we want to merge and defer on an alternative solution for later?